crates/ide_db/src/helpers.rs

```diff
@@ -51,7 +50,11 @@ pub fn get_path_in_derive_attr(
     if PathResolution::Macro(derive) != resolved_attr {
         return None;
     }
+    get_path_at_cursor(cursor)
+}
 
+pub fn get_path_at_cursor(cursor: &Ident) -> Option<ast::Path> {
```

Comment:
```suggestion
/// Parses the path the identifier is part of inside a token tree.
pub fn get_path_at_cursor_in_tt(cursor: &Ident) -> Option<ast::Path> {
```
As this is specifically for TokenTrees(macro input) we should clarify that with a more fitting name + comment.
Same for `parse_comma_sep_paths` below.